docs/core/compendium_de.md

```diff
@@ -0,0 +1,1177 @@
+Dogu-Client sie sonst als Anmeldedaten verwendet.
+
+Beispiel output:
+
+```
+echo "database: ${DATABASE}"
+echo "username: ${USER}"
+echo "password: ${PASSWORD}"
+```
+
+service-account-delete: Wenn ein Dogu gelöscht wird, sollte auch der verwendete Service-Account gelöscht werden. Aus
+diesem Grund muss das Dogu, das Service-Accounts erzeugt (wie postgresql), einen service-account-delete Befehl
+implementieren. Dieses Skript wird bei der Löschung eines Dogus ausgeführt. Der Dogu-Client übergibt den
+Namen des Service-Accounts als einzigen Parameter. Im Gegensatz zum Befehl service-account-create kann das Skript
+Logs ausgeben, da der Dogu-Client diese Daten nicht verwendet.
+
+Beispiel ExposedCommands für Service-Accounts:
+
+```
+"ExposedCommands": [
+   {
+     "Name": "service-account-create",
+     "Description": "Creates a new service account",
+     "Command": "/create-sa.sh"
+   },
+   {
+     "Name": "service-account-remove",
+     "Description": "Removes a service account",
+     "Command": "/remove-sa.sh"
+   }
+ ],
+```
+
+Benutzerdefinierte Befehle: Darüber hinaus kann ein Dogu Befehle angeben, die nicht automatisch im Lebenszyklus des
+Dogus ausgeführt werden (z.B. Upgrade). Zum Beispiel kann ein Dogu wie Redmine einen Befehl zum Löschen oder
+Installieren eines Plugins angeben. Zur Laufzeit kann ein Administrator diesen Befehl mit der cesapp wie folgt aufrufen:
+
+```
+cesapp command redmine plugin-install scrum-plugin
+```
+
+Beispiel:
+
+```
+"ExposedCommands": [
+   {
+     "Name": "plugin-install",
+     "Description": "Installs a plugin",
+     "Command": "/installPlugin.sh"
+   }
+ ],
+```
+
+### Volumes
+
+Volumes enthält eine Liste von [Volume](#type-volume), die [OCI container volumes](https://opencontainers.org/)
+definiert. Dieses Feld ist optional.
+
+Volumes werden während der Erstellung oder des Upgrades eines Dogus erstellt. Volumes bieten eine Leistungssteigerung im
+Vergleich zum Container-Speicher.
+
+Beispiele:
+
+```
+[ { "Name": "temp", "Path":"/tmp", "Owner": "1000", "Group": "1000", "NeedsBackup": false} ]
+```
+
+### HealthCheck
+
+HealthCheck definiert eine einzige Möglichkeit, den Zustand des Dogus auf Beobachtbarkeit zu prüfen. Dieses Feld ist
+optional.
+
+Veraltet: verwenden Sie stattdessen [HealthChecks](#healthchecks).
+
+### HealthChecks
+
+HealthChecks definieren eine Liste von [HealthCheck](#type-healthcheck). Sie bieten mehrere Möglichkeiten, den Zustand
+des Dogus auf Beobachtbarkeit zu prüfen. Dieses Feld ist optional.
+
+HealthChecks werden in verschiedenen Anwendungsfällen verwendet:
+
+- um die `dogu is starting` Seite anzuzeigen, bis das Dogu beim Start healthy ist
+- zur Überwachung über `cesapp healthy <dogu-name>`
+- für die Überwachung über das Admin-Dogu
+- um zu vermeiden, dass ein unhealthy Dogu gesichert wird
+
+Es gibt verschiedene Arten von HealthChecks:
+
+- state, über den `/state/<dogu>` Registry-Schlüssel, der von ces-setup gesetzt wird
```

Comment:
Argh, the base text is incomplete and misleading. `state` is way more often set/read by `doguctl` compared to `ces-setup`.

What do we do here?